_includes/manuals/1.23/1.23.0-RC1_release_notes.md

```diff
@@ -0,0 +1,303 @@
+
+#### Unattended installations
+* Allow HTTP for userdata controler ([#27055](https://projects.theforeman.org/issues/27055))
+* Invalid variable $release used in @initrd macro for FreeBSD ([#26949](https://projects.theforeman.org/issues/26949))
+* serve templates for cloud-init nocloud provider ([#24881](https://projects.theforeman.org/issues/24881))
+
+#### Users
+* Implement impersonate feature for easier debugging ([#26003](https://projects.theforeman.org/issues/26003))
+
+#### Web Interface
+* Datetime picker does not open with en_gb locale  ([#27312](https://projects.theforeman.org/issues/27312))
+* env variable to disable redux-logger ([#27105](https://projects.theforeman.org/issues/27105))
+* Editor replace toString with lodash ([#27104](https://projects.theforeman.org/issues/27104))
+* Migrate to @theforeman/vendor and @theforeman/vendor-dev pkg ([#27074](https://projects.theforeman.org/issues/27074))
+* Add onPopState HOC ([#27049](https://projects.theforeman.org/issues/27049))
+* Reuse Page Reducer Logic ([#27016](https://projects.theforeman.org/issues/27016))
+* extlogin has no styles ([#27008](https://projects.theforeman.org/issues/27008))
+* Fix typo in the main readme file ([#26931](https://projects.theforeman.org/issues/26931))
+* disable Turbolinks for 3rd party Routing ([#26854](https://projects.theforeman.org/issues/26854))
+* react-router foundation #4 ([#26805](https://projects.theforeman.org/issues/26805))
+* Create Statistics Route ([#26783](https://projects.theforeman.org/issues/26783))
+* react-router foundation #3 - Bookmarks ([#26753](https://projects.theforeman.org/issues/26753))
+* Add a dockerfile for foreman ([#26741](https://projects.theforeman.org/issues/26741))
+* Layout doesn't show active on nested routes ([#26712](https://projects.theforeman.org/issues/26712))
+* add react-router foundation #2 ([#26696](https://projects.theforeman.org/issues/26696))
+* Refactor folder structure of ToastList ([#26652](https://projects.theforeman.org/issues/26652))
+* unneeded asset compile for hidden_values.js and lookup_keys.js ([#26647](https://projects.theforeman.org/issues/26647))
+* Improve Layout active MenuItems performance ([#26581](https://projects.theforeman.org/issues/26581))
+* add react-router foundation ([#26186](https://projects.theforeman.org/issues/26186))
+* Vertical navigation missing IDs ([#26052](https://projects.theforeman.org/issues/26052))
+* Add components from katellos 'move_to_foreman' directory to foreman ([#25888](https://projects.theforeman.org/issues/25888))
+* Refactor the FactChart component structure to group files by domain ([#25864](https://projects.theforeman.org/issues/25864))
+* Move LoginPage into a React component using patternfly-react LoginPage component. ([#24990](https://projects.theforeman.org/issues/24990))
+* Move gridster from bundle to npm ([#16287](https://projects.theforeman.org/issues/16287))
+
+#### foreman-installer script
```

Comment:
These should move to the `#### Installer` section

Reply:
done!